src/pages/GetStarted/index.tsx

```diff
@@ -187,7 +187,6 @@ export class GetStarted extends React.PureComponent<Props, State> {
         </PageSection>
         <PageSection
           variant={PageSectionVariants.light}
-          padding={{ default: 'noPadding' }}
```

Comment:
Changing the Get Started page layout is not in the scope of this issue. Please revert this change, and if necessary provide another PR with proper motivation why do you want to do that.

Reply:
I experimented and forgot to return. Fixed.